homeassistant/components/light/zwave.py

```diff
@@ -217,11 +217,10 @@ def _get_color_values(self):
                 self._value_color = value_color
 
         if self._value_color_channels is None:
-            for value_color_channels in self._value.node.get_values(
-                    class_id=zwave.const.COMMAND_CLASS_SWITCH_COLOR,
-                    genre=zwave.const.GENRE_SYSTEM,
-                    type=zwave.const.TYPE_INT).values():
-                self._value_color_channels = value_color_channels
+            self._value_color_channels = self.value_handler(
```

Comment:
member should be dropped here, since it's expecting a reference to the Value object.